### DISCUSSION TITLE
Large Album Loading Issues

### DISCUSSION THREAD

**Discussion by Chips-OToole** [thumbsup/thumbsup#262] [Q&A]
Hello, anyone have any workarounds for loading large albums? Once my albums reach about 1500+ images the load times become unacceptable (and that's when loading locally). Anyway to implement 'lazy loading'?

Thanks,

Chips

**Comment by rprieto** [comment]
Hi, have you tried `--album-page-size` ? It doesn't fix loading times per se but it breaks up albums into smaller pages when needed.

Otherwise lazy loading is the way to go. I think the [flow](https://github.com/thumbsup/theme-flow/blob/master/theme/album.hbs) theme implements that. Note that once all images are loaded (after a lot of scrolling) the page might still be very slow... depending on wether the lazy-loading also unloads images as needed or not.
